releasenotes/notes/primitives-transpile-operator-afce8badf98e5869.yaml

```
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Add :func:`~primitives.utils.transpile_operator` function.
```

I'm not sure this will be found as it is, I think it would be safer to either add the full path, or use `.transpile_operator` only 🙂 
```suggestion
    Add :func:`~qiskit.primitives.utils.transpile_operator` function.
```